src/main/java/org/jabref/gui/collab/EntryChangeViewModel.java

```diff
@@ -39,6 +41,19 @@ public EntryChangeViewModel(BibEntry entry, BibEntry newEntry, BibDatabaseContex
 
     }
 
+    /**
+     *   We override this here to select the radio buttons accordingly
+     */
+    @Override
+    public void setAccepted(boolean accepted) {
+        super.setAccepted(accepted);
+        if (accepted) {
```

Comment:
This is no longer needed now, right?

Reply:
Nope, this is still needed. It ensures that when you click on "mark all changes" that the radio buttons on the right are selected and the merge entry will be the correct one.